src/ripe/brand.py

```diff
@@ -5,20 +5,20 @@ class BrandAPI(object):
 
     def config_brand(self, brand, model):
         url = self.base_url + "brands/%s/models/%s/config" % (brand, model)
-        contents = self.get(url, auth = False)
+        contents = self.get(url, auth = False, json = True)

```

Comment:
elaborate on this fix

Reply:
Without json = True, this url = self.base_url + "brands/%s/models/%s/validate" % (brand, model) wasn't calling the ripe-core route
@appier.route("/api/brands/<str:brand>/models/<str:model>/validate", "GET", json = True)
It was always exiting with a timeout.

Reply:
trust me, the `json = True` there has nothing to do with that error the @appier.route is a flag that controls the kind of serialization in case of error and other small details.